ci/install_travis.sh

```diff
@@ -114,6 +114,10 @@ if [ "$COVERAGE" ]; then
     pip install coverage pytest-cov
 fi
 
+if [ "$DOC_BUILD" ]; then
```

Comment:
move instead to ``ci/requirements-3.5_DOC_BUILD.sh`` (and I would install install via conda-forge channel (just like feather-format)